app/views/views.py

```diff
@@ -24,15 +25,26 @@ def __init__(self):
         super(QuestionsList, self).__init__()
 
     def post(self):
-        qtn_id = len(Questions)
-        qtn_id += 1
+        Question_ID = len(Questions)
```

Comment:
What happens if am to delete a question. Does the ID update, or would there be repeated ID's

Reply:
No it doesnt update. The ID of the remaining data stays the same